i18n/es.po

```diff
@@ -54,6 +54,9 @@ msgstr "Programa"
 msgid "Access to Template Generation"
 msgstr "Acceso a la generación de plantillas"
 
+msgid "Access to Import Data"
+msgstr "Importar datos"
```

Comment:
I guess it should be "Acceso a la importación de datos" or similar. Probably this came from a fuzzy translation, where it re-uses some (similar) existing translation.